solvebio/resource/datasetcommit.py

```diff
@@ -20,11 +21,21 @@ class DatasetCommit(CreateableAPIResource, ListableAPIResource,
         ('created_at', 'Created'),
     )
 
+    @property
+    def dataset_id(self):
+        return self['dataset']['id']
+
+    @property
     def dataset(self):
-        self._client.Dataset.retrieve(self['dataset'])
+        response = self._client.get(self['dataset']['url'], {})
```

Comment:
is dataset['url'] a relative URL or absolute?

Reply:
absolute

Reply:
I guess this is ok!